deployer.sh

```diff
@@ -163,7 +163,7 @@ RETRCODE=0
 
 function install_repo() {
     IP=${1}
-    sshpass -p"${ROOT_PASSWORD}" ssh ${SSH_ARGS} "root@${IP}" "curl  https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/master/origin.repo > /etc/yum.repos.d/origin-master.repo; yum update -y"
+    sshpass -p"${ROOT_PASSWORD}" ssh ${SSH_ARGS} "root@${IP}" "curl  https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/v3.10.0-alpha.0/origin.repo > /etc/yum.repos.d/origin-master.repo; yum update -y"
```

Comment:
Looks good, but we do need a way to pull the latest unreleased 3.10 @sdodson any ideas?

Reply:
There should be https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/release-3.10/origin.repo  but there isn't checking on that.

Reply:
Ok, that .repo file now exists. I'd use that.

Reply:
Thanks @sdodson!